src/components/AddRepsModal/InputModeSwitch.tsx

```diff
@@ -8,27 +8,50 @@ type Props = {
 };
 
 type ButtonProps = {
+  id?: string;
   buttonMode: InputMode;
   currentMode: InputMode;
   onClick: (mode: InputMode) => void;
 };
 
-class InputModeButton extends React.Component<ButtonProps> {
+type ButtonState = {
+  rpeTooltipIsOpen: boolean;
+};
+
+class InputModeButton extends React.Component<ButtonProps, ButtonState> {
+  public state: Readonly<ButtonState> = {
+    rpeTooltipIsOpen: false
+  };
+
   public render() {
     const active = this.props.buttonMode === this.props.currentMode;
     return (
-      <Button
-        color="primary"
-        size="sm"
-        className={!active ? "btn-primary--lighter" : ""}
-        onClick={this.handleClick}
-        outline={true}
-        active={active}
-      >
-        {this.props.children}
-      </Button>
+      <>
```

Comment:
Try building in production and see if it works with `<>` because they wrote in `create-react-app 2` that they added support for this syntax, so I assume there was no support before version 2.